backend/cesium_api/__init__.py

```diff
@@ -12,6 +12,7 @@
 DEFAULT_HOST = '0.0.0.0'
 DEFAULT_PORT = 9000
 DEFAULT_DATA_DIR = os.getenv("DATA_DIR", "../data")
+DEFAULT_TILES_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "geo-tiles"))
```

Comment:
```suggestion
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.